.travis/osx..script.sh

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
-if [ $QT5 ]; then
+if [ "$QT5" ]; then
         # Workaround; No FindQt5.cmake module exists
-        export CMAKE_PREFIX_PATH="$(brew --prefix qt55)"
+        CMAKE_PREFIX_PATH="$(brew --prefix qt55)"
```

Comment:
That seems to be `set -e` motivated.  More here:  http://stackoverflow.com/a/22636016/3196753

Since macOS [won't update to bash 4.0](https://blog.bennycornelissen.nl/bash-4-x-on-osx-the-bash-that-apple-wont-ship/), this is necessary, thanks.